readme.txt

```diff
@@ -1,8 +1,8 @@
 === Comet Cache ===
 
-Stable tag: 161119
+Stable tag: 161207
 Requires at least: 4.2
-Tested up to: 4.7
+Tested up to: 4.7-alpha
```

Comment:
@jaswsinc This should not be `4.7-alpha`, since `4.7` has been released. That's the reason this was changed. See https://github.com/websharks/comet-cache-pro/commit/34c3d1f4698b9432425845f501557ea26ce2a079.